scripts/python/junglecatsoftware/awstoolkit/outputFormatters.py

```diff
@@ -5,18 +5,19 @@ def printBigHeader( author = 'Jungle Cat Software', package = 'AWS-Toolkit', fun
   firstLineLen = len( firstLine )
   secondLine = function
   secondLineLen = len( secondLine )
-
+  
   longestLineLen = firstLineLen if firstLineLen >= secondLineLen else secondLineLen
   linePadLen = longestLineLen + 2
   headerLineLen = linePadLen + 2
 
-  print( ( '{0:*^' + str( headerLineLen ) + '}' ).format( '' ) )
-  print( ( '*{0: ^' + str( linePadLen ) + '}*' ).format( firstLine ) )
-  print( ( '*{0: ^' + str( linePadLen ) + '}*' ).format( secondLine ) )
-  print( ( '{0:*^' + str( headerLineLen ) + '}' ).format( '' ) )
+  print('*' * headerLineLen)
+  formatStr = '*{{0: ^{}}}*'.format(linePadLen)
+  print(formatStr.format(firstLine))
+  print(formatStr.format(secondLine))
+  print('*' * headerLineLen)
 
 def printSmallHeader( function = 'Output Formatter' ):
-  print( '\n==' + function + '==' )
+  print('\n=={}=='.format(function))
```

Comment:
I've always wondered about this kind of thing. Is there a reason to do it this way other than "consistency"? (Consistency is a good reason in and of itself, but I always wondered if there was more to it...)